load/dependencies.go

```diff
@@ -49,9 +50,9 @@ type Tracker[T comparable] interface {
 	GetObservedFailed() uint64
 }
 
-// Orchestrator executes the load test by coordinating the issuers to send
+// orchestrator executes the load test by coordinating the issuers to send
 // transactions, in a manner depending on the implementation.
-type Orchestrator interface {
+type orchestrator interface {
```

Comment:
I very well could be missing something here, but is this interface actually used anywhere other than for the compile time checks? An un-exported interface seems odd to me

Reply:
This is just for the compile time check that the 2 orchestrator implementations implement that interface.
When moving this to avalanchego, this can also stay unexported, given orchestrators will be defined locally there too.

What's wrong with unexported interfaces? We don't accept any argument with that interface, nor do we need to import it anywhere else

Reply:
Nothing wrong with it, and I agree that having it unexported rather than exported if it's not needed anywhere.

I guess my question is more turning that around more generally: if an interface is not used anywhere (internal or external to the package it's defined in), then what's the purpose of having it at all?

I'm fine with keeping it as is either way though.

Reply:
I'm fine with removing it, it's just more of a documentation for future orchestrators showing what minimal implementation an orchestrator should have.